README.md

```diff
@@ -11,19 +17,21 @@ Kotlin Multiplatform project repository template to set up all projects at [Nimb
 
 Please check [this](https://kotlinlang.org/docs/multiplatform-mobile-setup.html#install-the-necessary-tools) out for more details.
```

Comment:
We should also mention about:
```
Ruby 3.1.2
Xcode 13.3+
```

Reply:
@phongvhd93 we need Ruby to generate the iOS project or to run the final generated KMM project?

Reply:
@phongvhd93 I think we don't need to mention 
- Ruby, I can even run the script with my Ruby 2.6.10 🤷‍♂️ macOS default env has Ruby already.
- [Tuist](https://github.com/tuist/tuist) as the script will try to grab the missing for us.

Reply:
Updated in https://github.com/nimblehq/kmm-templates/pull/28/commits/ebc9518f2585cbd63933973fc0ba6f11c769b7ca